resources/lang/id/authorization.php

```diff
@@ -0,0 +1,160 @@
+                'no_forum_access' => 'Akses ke forum yang diminta diperlukan',
+                'not_owner' => 'Hanya pemilik topik yang dapat menghapus posting.',
+            ],
+
+            'edit' => [
+                'deleted' => 'Tidak bisa menyunting postingan yang telah dihapus',
+                'locked' => 'Topik telah dikunci untuk melakukan penyuntingan.',
+                'no_forum_access' => 'Akses ke forum yang diminta diperlukan.',
+                'not_owner' => 'Hanya pemilik topik yang dapat menyunting posting.',
+                'topic_locked' => 'Tidak dapat menyunting postingan di topik yang telah dikunci.',
+            ],
+
+            'store' => [
+                'play_more' => 'Cobalah bermain sebelum memposting di forum! Jika anda memiliki masalah tentang permainan, silakan posting ke Forum Bantuan dan Layanan Dukungan.',
+                'too_many_help_posts' => 'Anda harus memainkan permainan-nya lebih banyak sebelum dapat membuat postingan tambahan. Jika anda masih kesulitan memainkan permainan-nya, kirim email ke support@ppy.sh ', // FIXME: alamat email unhardcode
+            ],
+        ],
+
+        'topic' => [
+            'reply' => [
+                'double_post' => 'Anda baru saja memposting. Tunggu sebentar atau sunting posting terakhir Anda.',
+                'locked' => 'Tidak bisa membalas di topik yang telah dikunci.',
+                'no_forum_access' => 'Akses ke forum yang diminta diperlukan.',
+                'no_permission' => 'Tidak memiliki izin untuk membalas.',
+
+                'user' => [
+                    'require_login' => 'Silakan masuk untuk membalas.',
+                    'restricted' => 'Tidak dapat membalas saat dibatasi.',
+                    'silenced' => 'Tidak dapat membalas saat dibungkam.',
+                ],
+            ],
+
+            'store' => [
+                'no_forum_access' => 'Akses ke forum yang diminta diperlukan.',
+                'no_permission' => 'Tidak memiliki izin untuk membuat topik baru.',
+                'forum_closed' => 'Forum ditutup sehingga tidak dapat membuat postingan.',
+            ],
+
+            'vote' => [
+                'no_forum_access' => 'Akses ke forum yang diminta diperlukan.',
+                'over' => 'Polling selesai dan tidak dapat dipilih lagi.',
+                'voted' => 'Mengubah suara tidak diizinkan.',
+
+                'user' => [
+                    'require_login' => 'Silakan masuk untuk memilih.',
+                    'restricted' => 'Tidak dapat membalas saat dibatasi.',
+                    'silenced' => 'Tidak dapat membalas saat dibungkam.',
+                ],
+            ],
+
+            'watch' => [
+                'no_forum_access' => 'Akses ke forum yang diminta diperlukan.',
+            ],
+        ],
+
+        'topic_cover' => [
+            'edit' => [
+                'uneditable' => 'Sampul tidak valid ditentukan.',
```

Comment:
Umm, I don't really get the context here either but this should work better I suppose : 

+                'uneditable' => 'Gambar sampul yang Anda ingin terapkan tidak valid.',